src/components/WebCLI/WebCLI.tsx

```diff
@@ -35,16 +43,53 @@ const WebCLI = ({
 }: Props) => {
   const [connection, setConnection] = useState<Connection | null>(null);
   const [shouldShowHelp, setShouldShowHelp] = useState(false);
+  const [historyPosition, setHistoryPosition] = useState(0);
   const inputRef = useRef<HTMLInputElement>(null);
   const wsMessageStore = useRef<string>("");
   let [output, setOutput] = useState("");
   const sendAnalytics = useAnalytics();
   const storeState = useStore().getState();
+  const [cliHistory, setCLIHistory] = useLocalStorage<string[]>(
```

Comment:
We should consider adding a size limit to the size of history commands, where the size limit of the local storage is ~5-10 Mb which is huge however parsing and stringifying for example a 5Mb of JSON may not be ideal for performance.
having a limit around 200 items should be fine.

Reply:
Good idea! I've updated this.